cfgov/unprocessed/apps/filing-instruction-guide/css/main.less

```diff
@@ -61,6 +61,12 @@
     }
   }
 
+  .block__print-link {
```

Comment:
Kinda minor, but particularly in regard to global styles such as `block` we should probably name them for what the style is doing vs the context in which it is used. So something like `block__padded-irregular`.

Alternatively, this could be made to be a utility class for this specific use-case, e.g. `u-print-link` or `u-fig-print-link`.

Reply:
A utility class might be the way to go, as chaining this to `block` appears to imply that it may some day end up with the other `block`s in the DS, but this appears to be intended to be a focused style specific to FIG uses. If, however, it is intended to some day be apart of the DS `block` ecosystem, then chaining there makes sense.

Reply:
I'll switch it to a utility class, def makes more sense.